template_feed/Microsoft.DotNet.Web.ProjectTemplates.1.x/content/EmptyWeb-CSharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:55555/",
```

Comment:
Nit: this Url has a trailing slash, but the applicationUrl does not. The difference shouldn't affect anything, but we should be consistent.